### DISCUSSION TITLE
with 0.7.27 "no cert. or fingerprint" with static key

### DISCUSSION THREAD

**Discussion by peschau** [schwabe/ics-openvpn#1395] [General]
Hi,
I'm using a config with a static key for some times now, but with the last update there is only a message on the "generated config" tab stating: "an openVPN profile needs a CA certificate or a peer fingerprint"

Is the a way to get the last version (ok, I found several, but can not verify them) to be back in business until the cert is created and tested on my VPN gateway? Or is it possible to set a compatibility mode?

Regards Peter

**Comment by schwabe** [comment]
It is a bug, here is a workaround for now: http://plai.de/android/ics-openvpn-0.7.28pre.apk

Please not that static key support will be removed eventually.

**Reply by KeithMnemonic** [reply]
I tried this version and it still fails with the same error

**Comment by peschau** [comment]
Hi Arne,
works like a charm, thanks a lot for this really fast support.

Regards Peter
